torch_xla/runtime.py

```diff
@@ -131,6 +131,12 @@ def global_device_count() -> int:
   return len(torch_xla._XLAC._xla_get_all_devices())
 
 
+@requires_pjrt
+def local_runtime_devices() -> List[str]:
```

Comment:
@JackCaoG What do you think about this API? I wanted to include it so there's some way to discover the local device ID ordering.

Reply:
I think this could be handy.

Reply:
I'm removing this, we'll keep the new `_global_tensor_from_cpu_shards` private.